plugins/logitech-hidpp/fu-logitech-hidpp-device.c

```diff
@@ -88,19 +88,19 @@ static const gchar *
 fu_logitech_hidpp_device_get_icon(FuLogitechHidppDeviceKind kind)
 {
 	if (kind == FU_HIDPP_DEVICE_KIND_KEYBOARD)
-		return "input-keyboard";
+		return FU_DEVICE_ICON_INPUT_KEYBOARD;
 	if (kind == FU_HIDPP_DEVICE_KIND_REMOTE_CONTROL)
-		return "pda"; // ish
+		return FU_DEVICE_ICON_PDA; // ish
 	if (kind == FU_HIDPP_DEVICE_KIND_NUMPAD)
-		return "input-dialpad";
+		return FU_DEVICE_ICON_INPUT_DIALPAD;
 	if (kind == FU_HIDPP_DEVICE_KIND_MOUSE)
-		return "input-mouse";
+		return FU_DEVICE_ICON_INPUT_MOUSE;
 	if (kind == FU_HIDPP_DEVICE_KIND_TOUCHPAD)
-		return "input-touchpad";
+		return FU_DEVICE_ICON_INPUT_TOUCHPAD;
 	if (kind == FU_HIDPP_DEVICE_KIND_TRACKBALL)
-		return "input-mouse"; // ish
+		return FU_DEVICE_ICON_INPUT_MOUSE; // ish
 	if (kind == FU_HIDPP_DEVICE_KIND_PRESENTER)
-		return "pda"; // ish
+		return FU_DEVICE_ICON_PDA; // ish
 	if (kind == FU_HIDPP_DEVICE_KIND_RECEIVER)
 		return "preferences-desktop-keyboard";
```

Comment:
ohh if you're here, this should be `usb-receiver` now

Reply:
or `FU_DEVICE_ICON_USB_RECEIVER` rather.

Reply:
yes, that was what I wondered! I replaced all the 3 `preferences-desktop-keyboard` occurrences then :slightly_smiling_face: